.travis.yml

```diff
@@ -5,5 +5,12 @@ rust:
   - nightly
 
 script:
```

Comment:
Is it worth considering a build matrix that uses the toolchain rustfmt rather than stable only?